packages/world/ts/config/v2/namespaces.ts

```diff
@@ -23,7 +24,7 @@ export type validateNamespaces<namespaces, scope extends Scope = AbiTypeScope> =
   [namespace in keyof namespaces]: {
     [key in keyof namespaces[namespace]]: key extends "tables"
       ? validateTables<namespaces[namespace][key], scope>
-      : namespaces[namespace][key];
+      : ErrorMessage<`Expected full tables config.`>;
```

Comment:
This error isn't accurate. Maybe something like

```suggestion
      : ErrorMessage<`\`${key & string}\` is not a valid namespace config option.`>
```

Reply:
@ssalbdivad Do you happen to have a TS util to do something along the lines of `(keyof NamespacesInput).join(", ")`?

Reply:
@holic Yep, there is a type in `@arktype` util called `stringifyUnion`!